tokens/dist/json/tokens.json

```diff
@@ -4457,55 +4337,310 @@
       "value": "30px"
     }
   },
-  "component-pill-edge-to-text-100": {
-    "prop": "--spectrum-component-pill-edge-to-text-100",
+  "component-l-bold-font-family": {
+    "prop": "--spectrum-component-l-bold-font-family",
+    "ref": "var(--spectrum-sans-serif-font-family)",
+    "value": "Adobe Clean Spectrum VF"
+  },
+  "component-l-bold-font-family-font-size": {
+    "prop": "--spectrum-component-l-bold-font-family-font-size",
+    "ref": "var(--spectrum-font-size-200)",
     "desktop": {
       "value": "16px"
     },
     "mobile": {
-      "value": "20px"
+      "value": "19px"
     }
   },
-  "component-pill-edge-to-text-200": {
-    "prop": "--spectrum-component-pill-edge-to-text-200",
+  "component-l-bold-font-family-font-size-font-weight": {
```

Comment:
These are not quite right yet. TODO: need to find out why the compound variables are generating compound variable names! :)